content/news/playtest-20160930.md

```diff
@@ -0,0 +1,33 @@
+We are pleased to release the second playtest of this development cycle!
```

Comment:
Can we please include a second sentence here like "This refines and fixes the many new features that we added in [playtest-20160905](/news/playtest-20160904/) in preparation for a final release."?   We should make it easy for people to see the other changes compared to the last release.